src/main/scala/com/github/unchama/seichiassist/subsystems/mebius/bukkit/listeners/MebiusRenamePreventionListener.scala

```diff
@@ -45,4 +45,23 @@ class MebiusRenamePreventionListener extends Listener {
       cancelEventAndNotifyTheAlternative(event)
     }
   }
+
+  // 数字キーでホットバーのアイテムを金床のスロットに移動している時
+  @EventHandler def onSwap(event: InventoryClickEvent): Unit = {
+    // 金床を開いていない場合終了
+    if (!event.getView.getTopInventory.isInstanceOf[AnvilInventory]) return
+
+    val action = event.getAction
+    // event.getActionが数字キーによるアイテムの移動ならば処理を続行
+    if (action == null || (action != InventoryAction.HOTBAR_SWAP && action != InventoryAction.HOTBAR_MOVE_AND_READD)) return
+
+    val keyNum = event.getHotbarButton
+    val viewers = event.getViewers
+    // 押された数字キーに対応するホットバーのアイテムを取得する
+    Option(viewers.iterator().next().getInventory.getItem(keyNum)).foreach { item =>
+      // 金床の一番左のスロットにMebiusを移動させた場合
+      if (BukkitMebiusItemStackCodec.isMebius(item) && event.getView.convertSlot(0) == 0 && event.getRawSlot == 0)
+        cancelEventAndNotifyTheAlternative(event)
+    }
```

Comment:
```suggestion
    // 金床の一番左のスロットにアイテムを移動させた場合以外 return
    if (event.getView.convertSlot(0) != 0 || event.getRawSlot != 0) return
    
    // Mebiusを移動させた場合
    val bottomInventory = event.getView.getBottomInventory
    if (BukkitMebiusItemStackCodec.isMebius(bottomInventory.getItem(keyNum))) {
      cancelEventAndNotifyTheAlternative(event)
    }
```

内側の条件式を外に出して getBottomInventory した方が分かりやすいかも？